vault/logical_system.go

```diff
@@ -1309,6 +1317,19 @@ func (b *SystemBackend) handleMountTuneWrite(ctx context.Context, req *logical.R
 	return b.handleTuneWriteCommon(ctx, path, data)
 }
 
+func augmentTuneWriteParams(paramIn interface{}) ([]string, error) {
+	newVal := paramIn.([]string)
```

Comment:
This will panic if paramIn isn't a string slice.

Reply:
Ah, yeah, I will do the type assertion before invoking this!